src/Fixer/Casing/ConstantCaseFixer.php

```diff
@@ -110,12 +110,33 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
             if (
                 $this->isNeighbourAccepted($tokens, $tokens->getPrevMeaningfulToken($index))
                 && $this->isNeighbourAccepted($tokens, $tokens->getNextMeaningfulToken($index))
+                && !$this->isEnumCaseName($tokens, $index)
             ) {
                 $tokens[$index] = new Token([$token->getId(), $fixFunction($token->getContent())]);
             }
         }
     }
 
+    private function isEnumCaseName(Tokens $tokens, int $index): bool {
+        if (!\defined('T_ENUM')) { // @TODO: drop condition when PHP 8.1+ is required
```

Comment:
annotations are not supported in standard comments, you'll need to use a doc comment.

Reply:
In all fairness, this is how we did all the `@TODO` 's related to `enum`, so for me it is fine. Thing is with PHPDoc is that what is annotated here is not a structural element so is like to be changed into a comment by one of the rules. So I'm not against changing it, yet as it is now is also fine by me.

Reply:
Yep, this was copy/pasta from another line that did this same check, so let's leave as-is. Someone can do a find/replace if they care to update all locations.